db/functions/report_ticket_sales_by_organization.sql

```diff
@@ -0,0 +1,36 @@
+DROP FUNCTION IF EXISTS report_ticket_sales_by_organization(UUID, TIMESTAMP, TIMESTAMP);
+CREATE OR REPLACE FUNCTION report_ticket_sales_by_organization(p_organization_id UUID,  p_start_date TIMESTAMP, p_end_date TIMESTAMP) returns table  (
+    id uuid,
+    first_name text,
+    last_name text,
+    email text,
+    events int8,
+    tickets int8,
+    transferred int8,
+    gross_spent numeric
+)
+AS $$
+BEGIN
+    return query SELECT u.id,
+                        u.first_name,
+                        u.last_name,
+                        u.email,
+                        count(DISTINCT e.id)                                                               AS events,
+                        sum(CASE WHEN oi.item_type = 'Tickets' THEN 1 ELSE 0 END)                          AS tickets,
+                        sum(CASE WHEN oi.item_type = 'Tickets' AND ti.wallet_id != w.id THEN 1 ELSE 0 END) AS transferred,
+                        sum(oi.unit_price_in_cents * oi.quantity)                                          AS gross_spent
```

Comment:
Because there will be a ticket_instance row that will always link back to this order_item row we don't need to multiply by the quantity
```suggestion
                        sum(oi.unit_price_in_cents)                                          AS gross_spent
```